### ISSUE TITLE
DIY kit availability

### ISSUE THREAD

**Issue by daxliniere** [FOSV/Fuji-Atom-Interface#2]
Hi @FOSV,
Did you ever make these available as a DIY kit?

**Comment by FOSV** [comment]
No, I'm sorry but since BoM is available and gerbers too, you can order a batch from JLCPCB and share it with anyone else who needs it.

**Comment by daxliniere** [comment]
Thanks @FOSV

Anyone want one?

**Comment by balouken** [comment]
It would be nice if we could order them from JLCPCB soldered and ready to use 👍

**Comment by pauleec** [comment]
I would like to find a way I can have batches of these made up / soldered and ready to use. In Australia we either have the Fuj Wall controllers in most apartments or the Daikin. There is an excellent similar project to this for Daikin controllers called "FaikinHK" which does the same as this project for Daikin Wall controllers. In Australia we have an entrepreneurial guy already selling PCB pre-built/boxed units on Amazon and eBay. I'm interested in offering a similar service for Fujitsu heatpump owners/users. At the moment I have purchased and built the circuits for the unreality/fujihk project but have been experiencing the same issue reported after flashing the M5Atom with it not restarting/giving a way for the unit to be setup (no flashing LED) making the M5ATOM look like it is unresponsive/dead. If your boards/setup have resolved this I would be very interested in trying them out with a view of getting lots more to help others looking to get this working with their fujitsu heat pumps. 

With this in mind I have some questions:

1) Can JLCPCB build/solder the whole pcb with the required components? If no, is there a way to get hold of a completed PCB with components for testing? Where can the completed PCB be purchased from?
2) Is there a JLCPCB design that will work with the components recommended in the unreality/FUJIHK project? Has anyone got the circuit working with the M5ATOM, the M5ATOM S3 and other ESP32 boards (like the ESP32 C3 Mini)
3) I notice that your circuit is a little more complex than the one provided in the unreality/FujiHK project. Why is this?

Thanks for posting such a great project on Github. Its a terrific piece of work!

**Comment by daxliniere** [comment]
Hey @pauleec, fellow Aussie here, though now in London.

I am looking into getting a batch made. Would you like me to put you down for one or two?

Regarding question 3, I believe it's because the others you've seen are simply interface boards for another controller whereas this one is a self-contained satellite device which can connect to HomeAssistant directly.

**Comment by pauleec** [comment]
Thanks for getting back to me …

Yes please make a couple for me. I need the HomeKit version (not the one
for home assistant. I’ve been struggling getting the unreality/fujihk
circuit and m5atom .bin file working tho. Have you had any luck? I’d like
to keep stock of the Faikin board, a board to work with Fuji heat pumps if
possible. There are so many of them here in Australia I’m sure they will be
in high demand if they work well.

**Comment by pauleec** [comment]
Hi,

I managed to get the M5Atom running with FujiHK by compiling the C++ source
in Arduino. I had to downgrade the esp32 board library from 3.2.0 to 3.1.3
and copy the FujiHeatPump and HomeSpan libraries to your local ide library
to get it to compile. I was not able to get Arduino to compile the sketch
for the M5AtomS3 and also a cheap esp32-C3 dev board I bought off
AliExpress. Has anyone managed to get their Arduino IDE to compile and
upload the image to any other esp32 board and if so are their any pointers
on what tweaks and code levels I need to setup for the sketch to compile
and upload correctly? I’m keen to get other esp32 boards working with the
code.

On a slightly different note I’ve been working with 2 different apartment
Fujitsu Heat pump ac units. One unit has the older 3 wire attached wall
controller however the other is only wired with 2 of the 3 available wires
which feed to the backend heat pump. Do the 2 wire units use just gnd and
 sig as both the signal and power for the controller? Can I take my 12v
supply feed from the Sig feed as well (which in turn will drop the voltage
to 3.3v in my circuits case)? I have attached a few pictures for reference.

If you are ordering a few of your awesome PCBs will they also have the
components installed? How much will I need to send you for each one (keen
to get 5 to 10 if you can get them).

Is there anyone in your network of contacts that has got this
compiled/uploaded and working on other esp32 boards like the M5AtomS3 you
can connect me with? If there is a difference with the 2 wire controllers
is anyone in your network working on backward engineering the protocol so
the solution can work with the newer 2 wire wall controllers as well? Can
you connect me with them?

Thanks and huge cudos to all your work with the PCB design. It looks very
good. Has anyone created a 3d printable enclosure for it yet?

All the best

Paulee’s

**Comment by RXM307** [comment]
> I would like to find a way I can have batches of these made up / soldered and ready to use. In Australia we either have the Fuj Wall controllers in most apartments or the Daikin. There is an excellent similar project to this for Daikin controllers called "FaikinHK" which does the same as this project for Daikin Wall controllers. In Australia we have an entrepreneurial guy already selling PCB pre-built/boxed units on Amazon and eBay. I'm interested in offering a similar service for Fujitsu heatpump owners/users. At the moment I have purchased and built the circuits for the unreality/fujihk project but have been experiencing the same issue reported after flashing the M5Atom with it not restarting/giving a way for the unit to be setup (no flashing LED) making the M5ATOM look like it is unresponsive/dead. If your boards/setup have resolved this I would be very interested in trying them out with a view of getting lots more to help others looking to get this working with their fujitsu heat pumps.
> 
> With this in mind I have some questions:
> 
> 1. Can JLCPCB build/solder the whole pcb with the required components? If no, is there a way to get hold of a completed PCB with components for testing? Where can the completed PCB be purchased from?
> 2. Is there a JLCPCB design that will work with the components recommended in the unreality/FUJIHK project? Has anyone got the circuit working with the M5ATOM, the M5ATOM S3 and other ESP32 boards (like the ESP32 C3 Mini)
> 3. I notice that your circuit is a little more complex than the one provided in the unreality/FujiHK project. Why is this?
> 
> Thanks for posting such a great project on Github. Its a terrific piece of work!

While it appears JLCPCB will make these up, they ask for a BOM and a CPL in the following format in excel or csv
**BOM**
<google-sheets-html-origin>
Comment | Designator | Footprint | JLCPCB Part #（optional）
-- | -- | -- | --
IC TRANSCEIVER HALF 1/1 8DFN | IC1 | MCP2025-330E/MD |  
DC DC CONVERTER 0.9-6V | U1 | TPS82140SILR |  
Capacitor 10UF 16V X5R 0805 | C1 | C2012X5R1C106M085AC |  
Capacitor 22UF 16V X5R 0805 | C2 | CL21A226MOQNNNE |  
Capacitor 10UF 16V X5R 0805 | C3 | C2012X5R1C106M085AC |  
Resistor SMD 133K OHM 0.1% 1/4W 0805 | R1 | 2176094-7 |  
Resistor SMD 24.9K OHM 0.1% 1/4W 0805 | R2 | 4-2176093-0 |  
Resistor SMD 10K OHM 0.1% 1/8W 0805 | R3 | RG2012P-103-B-T5 |  
Resistor SMD 500 OHM 0.05% 1/8W 0805 | R4 | TNPU0805500RAZEN00 |  
LED GREEN CLEAR 0805 SMD | L1 | 150080VS75000 |  
Connector HEADER SMD 5POS 2.54MM | J1 | TSM-105-01-F-SV |  
Connector HEADER SMD 4POS 2.54MM | J2 | TSM-104-01-F-SV |  
Connector HEADER SMD R/A 3POS 1.5MM | J3 | S3B-ZR-SM4A-TF(LF)(SN) |  



**CPL** Sample only as I have no idea how to fill this out.....
<google-sheets-html-origin>
Designator | Mid X | Mid Y | Layer | Rotation
-- | -- | -- | -- | --
C1 | 95.0518mm | 22.6822mm | Top | 270
C2 | 106.4056mm | 23.0124mm | Top | 90
C3 | 109.0726mm | 22.8854mm | Top | 270
R2 | 109.5996mm | 16.6443mm | Bottom | 0



PCBway seem to be less restrictive but they asked if there should be a solder mask layer on the bottom?

PCBway look like you can share designs also for easy ordering, minimum order appears to be 5 though same as JLCPCB so I have have some spares if they are not cost prohibitive..... will let you know

**Comment by pauleec** [comment]
Did you read my post here.... seems there is an issue with the version of the compiler for the m5Atom and this code. You have to downgrade it. Read my post here. https://github.com/unreality/FujiHK/issues/7#issuecomment-2862044903

**Comment by RXM307** [comment]
> Did you read my post here.... seems there is an issue with the version of the compiler for the m5Atom and this code. You have to downgrade it. Read my post here. [unreality/FujiHK#7 (comment)](https://github.com/unreality/FujiHK/issues/7#issuecomment-2862044903)

I'm still sourcing the hardware I haven't started with firmware yet

**Comment by a10kiloham** [comment]
FYI this works fine now in PlatformIO with the following tweak:
https://github.com/a10kiloham/FujiHK/blob/master/platformio.ini

Builds fine. I didn't try the access point via Wifi, but just went into the serial console in Platformio and then set the wifi there. I also reset the setup code then it added fine in Home Assistant after that.

**Comment by Chrix0689** [comment]
> Hi [@FOSV](https://github.com/FOSV), Did you ever make these available as a DIY kit?

Hi All, is there any of these available to be bought? Im in Aus and if there is someone who has rather than me trying to get some built would be much appreciated.  Happy to pay.

Thanks 
Chris
